book/src/performance.md

```diff
@@ -24,6 +24,26 @@ monomorphised code. On average, expect `.to_string()` to be 100% to 200% slower
 [`.to_string()`]: <https://doc.rust-lang.org/stable/std/string/trait.ToString.html#tymethod.to_string>
 [`format!()`]: <https://doc.rust-lang.org/stable/std/fmt/fn.format.html>
 
+## Faster Rendering of Custom Types
+
+Every type that implements [`fmt::Display`] can be used in askama expressions: `{{ value }}`.
+Rendering with `fmt::Display` can be slow, though, because is uses [dynamic methods calls] in its
+[`fmt::Formatter`] argument. To speed up rendering (by a lot, actually),
+askama adds the trait [`FastWritable`]. For any custom type you want to render,
+it has to implement `fmt::Display`, but if it also implements `FastWritable`,
```

Comment:
Might be worth explaining how askama does that, what do you think?

Reply:
Makes sense! I added a link. [Autoref-based specialization](https://lukaskalbertodt.github.io/2019/12/05/generalized-autoref-based-specialization.html) is such a strange language feature that others should know it, too. :)

Reply:
It's indeed a very nice feature! :)